tests/testthat/test-backend-mssql.R

```diff
@@ -200,3 +89,52 @@ test_that("custom lubridate functions translated correctly", {
   expect_equal(trans(quarter(x, with_year = TRUE)), sql("(DATENAME(YEAR, `x`) + '.' + DATENAME(QUARTER, `x`))"))
   expect_error(trans(quarter(x, fiscal_start = 5)))
 })
+
+with_options(dbplyr_table_num = 0, verify_output("sql/mssql.txt", {
```

Comment:
The structure of these should be:

```
test_that("...", {
  local_options(dbplyr_table_num = 0)
  verify_output(test_path("path"), {
     ...
  }
}
```

That way each test follows the same form, works interactively and from `devtools::test()`, and is easily understood at a glance. 

Copy and paste is much more acceptable in tests because it's more important than each test be independent and easily understood, than have the smallest amount of code possible.